### ISSUE TITLE
How can we optimise work on paid / priority features/hot fixes?

### ISSUE THREAD

**Issue by sdepold** [sequelize/meetings#26]

**Comment by WikiRik** [comment]
I think it would help if when making the PR it is clear that we do it because of a bug fix/new feature requested on OpenCollective. That would probably up the priority of possible reviewers. This could be included in the PR template.

Also it might help to request reviews of a larger group of people, like `@ sequelize / maintainers` (additional spaces here to not ping people)

**Comment by sdepold** [comment]
It seems that mentioning people doesn't have a too positive effect. What do you think about adding a label (priority or something) to the relevant issues / prs?

**Comment by WikiRik** [comment]
A label can't hurt to add

**Comment by ephys** [comment]
> What do you think about adding a label (priority or something) to the relevant issues / prs?

`funded`?

**Comment by WikiRik** [comment]
I agree, let's add `funded` and add to the contributing docs that working on the issue makes you eligible for compensation.
